README.md

```diff
@@ -93,6 +93,18 @@ The following changes are now available in production:
 ### individuals-reliefs-api
 
 ### individuals-state-benefits-api
+The following changes are now available in production:
+* Some error response status codes corrected from 403 to 400, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#10th-january-2023).
+* Added new `RULE_TAX_YEAR_NOT_SUPPORTED` error, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#10th-january-2023).
```

Comment:
This link currently links to the 403 to 400 changes, did you mean to link to the other entry (https://github.com/hmrc/income-tax-mtd-changelog/wiki#10th-january-2023-1) (notice the `-1` on the end). However, that change is to update the message and description of the error, as opposed to adding it.

If there are endpoints where it has been added, they should probably be mentioned.

Reply:
I realised the error was added as part of the TYS implementation, so I can remove this line